### PR TITLE
add: list users at the bottom

### PR DESCRIPTION
**Pull request by koverchik**

### CODE REVIEW

File: agoraStreaming/src/Components/ListUsers/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  const {hiddenUsers} = props;
+
+  return (
+    <View style={styles.container}>
+      <FlatList
+        horizontal
+        data={hiddenUsers}
+        renderItem={({item}) => {
+          return (
+            <View style={styles.pointUserName} key={'pointUser' + item.uid}>
```
Comment (DimaKelek) [inline agoraStreaming/src/Components/ListUsers/index.tsx:16]: remove 'key'

**Review by DimaKelek — APPROVED** [review]

File: agoraStreaming/src/Components/ListUsers/index.tsx (hunk 1 above)
Comment (koverchik) [reply agoraStreaming/src/Components/ListUsers/index.tsx:16]: Ok, I removed prop key

**Review by DimaKelek — APPROVED** [review]

File: agoraStreaming/src/Components/ListUsers/styles.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+};
+const SIZE_USER_POINT = 40;
+
+export const styles = StyleSheet.create<UserNameLabelType>({
+  pointUserName: {
+    height: SIZE_USER_POINT,
+    width: SIZE_USER_POINT,
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: 20,
```
Comment (evgenyshenets91) [inline agoraStreaming/src/Components/ListUsers/styles.ts:15]: SIZE_USER_POINT/2

File: agoraStreaming/src/Components/ListUsers/styles.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    backgroundColor: '#5ebf9d',
+    margin: 10,
+  },
+  container: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
```
Comment (evgenyshenets91) [inline agoraStreaming/src/Components/ListUsers/styles.ts:21]: it is not necessary

File: agoraStreaming/src/Screens/Live/index.tsx [hunk 4]
```diff
@@ -50,9 +56,36 @@ export const Live: FC<LiveScreenProps> = (props) => {
...
-    AgoraEngine.current?.leaveChannel();
-    navigation.goBack();
+  const callbackFunctionUserOffline = (uid: number) => {
+    setPeerIds((prev) => prev.filter((userData) => userData.uid !== uid));
+  };
+  const callbackUserMuteAudio = (uid: number, muted: boolean) => {
+    setPeerIds((prevState) => {
+      return mute({uid, muted, device: 'voice'}, prevState);
+    });
+  };
```
Comment (evgenyshenets91) [inline agoraStreaming/src/Screens/Live/index.tsx:66]: add empty lines between functions

File: agoraStreaming/src/Components/ListUsers/styles.ts (hunk 3 above)
Comment (koverchik) [reply agoraStreaming/src/Components/ListUsers/styles.ts:21]: I deleted this property style.

File: agoraStreaming/src/Components/ListUsers/styles.ts (hunk 2 above)
Comment (koverchik) [reply agoraStreaming/src/Components/ListUsers/styles.ts:15]: I changed this property. Thanks so much for your tip.

**Pull request merged**